src/templates/static.js

```diff
@@ -127,16 +127,21 @@ const components = {
   GhostCard,
 }
 
-const StaticPage = ({ data: { mdx } }) => {
+const StaticPage = ({ data: { siteData, mdx }, pageContext }) => {
   const intl = useIntl()
   const isRightToLeft = isLangRightToLeft(intl.locale)
-  const tocItems = mdx.tableOfContents.items
 
-  // TODO some `gitLogLatestDate` are `null` - why?
+  console.log({ mdx })
```

Comment:
(Intentional?)

Reply:
Removing the comment, yes. Adding a console.log, no 😳